docs/vectorizer/api-reference.md

```diff
@@ -81,6 +84,21 @@ Vectorizer offers the following APIs:
   performance of vectorizers.
 
 
+## Install or upgrade the database objects necessary for vectorizer
+
+You can install or upgrade the database objects necessary for vectorizer by running the following python code:
+
+```python
+import pgai
+
+pgai.install(DB_URL)
+```
```

Comment:
```suggestion
```python
import pgai

pgai.install(DB_URL)
```
Alternatively this cli command:
```bash
pgai install -d DB_URL
```

Reply:
Definitely the cli command looks the defacto way for the users